rllib/evaluation/worker_set.py

```diff
@@ -330,10 +330,11 @@ def recreate_failed_workers(self) -> Tuple[List[ActorHandle], List[ActorHandle]]
                 config=self._remote_config,
             )
             # Sync new worker from local one.
-            new_worker.set_weights.remote(
```

Comment:
should try_recover_from_step_attempt() take a local worker as parameter so it can always sync weights?